<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1870</width>
    <height>1099</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btn_addNode">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>120</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>addNode</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btn_batchNodes">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>260</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>batch place nodes</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btn_addRouteControl">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>170</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>addRouteMode</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>320</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>saveMap</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>360</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>loadMap</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btn_startSim">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>410</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>startSimulation</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btn_simControl">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>450</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PAUSE</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btn_changeAttr">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>210</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>changeAttr</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1870</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btn_addNode</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addNode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1500</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>860</x>
     <y>564</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_batchNodes</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>batchAddingNodes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1500</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>1083</x>
     <y>395</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveMap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1500</x>
     <y>383</y>
    </hint>
    <hint type="destinationlabel">
     <x>730</x>
     <y>461</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>loadMap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1500</x>
     <y>412</y>
    </hint>
    <hint type="destinationlabel">
     <x>1072</x>
     <y>587</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_startSim</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>startSim()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1500</x>
     <y>471</y>
    </hint>
    <hint type="destinationlabel">
     <x>793</x>
     <y>381</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_changeAttr</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>changeAttribution()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1598</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>1273</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addNode()</slot>
  <slot>batchAddingNodes()</slot>
  <slot>saveMap()</slot>
  <slot>loadMap()</slot>
  <slot>startSim()</slot>
  <slot>changeAttribution()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
